db/model/botData.js

```diff
@@ -162,15 +162,22 @@ module.exports = function botData(seq, dataTypes) {
       foreignKey: 'roomId',
       allowNull: false,
     });
+
     assoc.room = BotData.belongsTo(models.Bot, {
       foreignKey: 'botId',
       allowNull: false,
     });
+
+    assoc.createdBy = BotData.belongsTo(models.User, {
+      foreignKey: 'createdBy',
```

Comment:
also add this so that it's compatible with all the other resource types in refocus:

`as: 'user'`